### PR TITLE
Added Login and Signup forms

### PR DESCRIPTION
**Pull request by mykytazaginei**
added Login and Signup without styles forms

### CODE REVIEW

**Review by dariia-kachalova — COMMENTED** [review]
Looks ok

File: react-portal-employees/src/components/SignUp.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          <form>
+            <div className=''>
+                <label className=''>Login</label>
+                <input type='login'/>
+            </div>
+            <div className=''>
+                <label className=''>Password</label>
+                <input className='' type='password'  />
+            </div>
+            <div className=''>
```
Comment (DmytroDorodnykhTeamInternational) [inline react-portal-employees/src/components/SignUp.js]: why empty className everywhere?

**Review by DmytroDorodnykhTeamInternational — APPROVED** [review]

File: react-portal-employees/src/components/SignUp.js (hunk 1 above)
Comment (mykytazaginei) [reply react-portal-employees/src/components/SignUp.js]: I originally did with the style, I forgot to delete, I'll fix it

**Review by Ygoraphael — APPROVED** [review]
Looks nice, i think you just should create a new branch for each task.

**Review by VladyslavCherednychenko — APPROVED** [review]

**Pull request merged**